server/re/src/irods_re_serialization.cpp

```diff
@@ -1019,7 +1076,9 @@ namespace irods {
                 { std::type_index(typeid(rodsObjStat_t*)), serialize_rodsObjStat_ptr },
                 { std::type_index(typeid(genQueryInp_t*)), serialize_genQueryInp_ptr },
                 { std::type_index(typeid(genQueryOut_t*)), serialize_genQueryOut_ptr },
-                { std::type_index(typeid(char**)), serialize_char_ptr_ptr }
+                { std::type_index(typeid(char**)), serialize_char_ptr_ptr },
+		{ std::type_index(typeid(openedDataObjInp_t*)), serialize_openedDataObjInp_ptr },
+		{ std::type_index(typeid(openedDataObjInp_t**)), serialize_openedDataObjInp_ptr_ptr }
```

Comment:
indentation

Reply:
fixed emacs indentation for spaces instead of tabs, done.